docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -40,25 +40,28 @@ If you prefer a step-by-step installation process, complete the following steps
 
     ```bash
     sudo apt-get update
-    sudo apt-get install ca-certificates curl apt-transport-https lsb-release gnupg
+    sudo apt-get install apt-transport-https ca-certificates curl gnupg lsb-release
     ```
 
 2. Download and install the Microsoft signing key:
 
     ```bash
     sudo mkdir -p /etc/apt/keyrings
-    curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-        gpg --dearmor |
-        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
+    curl -sL https://packages.microsoft.com/keys/microsoft.asc |\
```

Comment:
```suggestion
    curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
```
`\` is not necessary as the line endswith `|`

Reply:
can switch the curl arguments, but why not keep the `| \` which would support multiline paste?

Reply:
The command also works without `\`.  Could you provide an example of how it fails?

Reply:
https://www.gnu.org/software/bash/manual/bash.html#Escape-Character.
I'm nitpicking I guess but same with the `{}` above, why not follow the standards?

Reply:
While there's no official documentation on this, `\` seems to be redundant if a line ends with `|`, `&&`, or `||`.

Ref:
https://stackoverflow.com/a/6714441/2251785
https://unix.stackexchange.com/questions/281309/shell-syntax-how-to-correctly-use-to-break-lines